src/api/healthcare/api.js

```diff
@@ -1,7 +1,10 @@
-export default (config, fetch) => fetch(config.apiUrl).then(response => {
-    if (response.status !== 200) {
-        return false;
-    }
+export default (config, fetch) =>
```

Comment:
You may not agree, but expanding `config` parameter makes the function signature explicit about what are the required parameters.

```js
export default ({ apiUrl, endPoint }, fetch) =>
```